frontend/src/app/components/Chrome/Header/User.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import { Link } from "react-router-dom";
+
+export default class Brand extends Component {
```

Comment:
Should this be `User` instead of `Brand`?

Reply:
yep this is what Michael talked about the explicit exports, I have changed it now to be explicit so this kind of typos don't happen in these two classes at least. Will work to do the same in all files eventually.